src/templates/blog-post.tsx

```diff
@@ -44,6 +47,13 @@ const BlogArticleTemplate: React.FC<BlogArticleTemplateProps> = ({ data }) => {
             {data.markdownRemark.frontmatter.title}
           </BlogPostTitle>
           <Markdown data={data.markdownRemark.rawMarkdownBody} />
+          {data.markdownRemark.frontmatter.author && (
+            <Byline
+              author={data.markdownRemark.frontmatter.author}
+              date={new Date(data.markdownRemark.frontmatter.date)}
```

Comment:
please use https://date-fns.org/v2.15.0/docs/parseISO for that, as our date string is not 100% compatible with ISO and the date-fns methods catches that